src/Context/ApiContext.php

```diff
@@ -85,6 +85,16 @@ public function theRequestHeaderContains(string $header, string $value): void
         $this->headers[$header] = $processedHeader;
     }
 
+    /**
+     * @Given the :headerName request header contains multiline value:
+     */
+    public function theRequestHeaderContainsMultiline(string $header, PyStringNode $params): void
+    {
+        $processedParams = $this->stringManager->substituteValues($this->savedValues, trim($params->getRaw()));
```

Comment:
These are just my thoughts. Maybe need to add trim per each line?

Reply:
After discussion we decided to postpone this feature